config/wikis/trwiki.yaml

```diff
@@ -3,25 +3,48 @@ label: Turkish Wikipedia
 host: tr.wikipedia.org
 
 external_samples:
-  sampled_revisions.20k_2015:
-    quarry_url: http://quarry.wmflabs.org/run/168286/output/0/json-lines?download=true
```

Comment:
Why remove this section?  We should probably keep it.

Reply:
Thanks, I've kept this.